server/server.go

```diff
@@ -43,28 +43,30 @@ func New(opts ...options.Modifier) (*Server, error) {
 		appSource = client.NewHTTPSource(vOpts.ControlPlane, nil)
 	}
 
+	syncer := syncer.New(vOpts, appSource)
```

Comment:
I think it would improve the readability of the code a bit if we instead construct the syncer in start.go and pass it into the server and backend. 

That makes it super clear that the syncer is an in-memory appsource and that its shared between the front and back of the house. 

Right now its a bit awkward because I have to go into the server code to figure out what a syncer is and why I care to extract it from the server to inject it into the backend.